requirements.pip

```diff
@@ -0,0 +1,6 @@
+Django==1.8.5
+jsonfield==1.0.3
+djangorestframework==3.3.0
+git+https://github.com/alanjds/drf-nested-routers@8c23e2143be86733e833ff25f88f1fb41a26155f#egg=drf-nested-routers
```

Comment:
There is a yet-unreleased fix which is awaiting a release.

https://github.com/alanjds/drf-nested-routers/issues/55